File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.go
func productExceptSelf(nums []int) []int {
    res := make([]int, len(nums))
    for i,_ := range nums {
    res[i] = 1
        for j:=0; j < len(nums); j++ {
            if j != i {
                res[i] = res[i] * nums[j]
            } 
        }
    }
    return res
}

File: Data Structures & Algorithms/binary-search/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number}
     */
    search(nums, target) {
        let l = 0
        let r = nums.length-1
        let res = -1
        while (l<=r){
            const mid = Math.trunc((l+r)/2)
            if (nums[mid]=== target) {
                return mid
            } else if (nums[mid] < target) {
                l = mid +1
            } else {r = mid-1}
        }

        return res
    }
}
